src/Appwrite/Utopia/Request/Filters/V12.php

```diff
@@ -0,0 +1,156 @@
+                $content = $this->convertQueries($content);
+                break;
+            case "database.createCollection":
+                $content = $this->addId($content, 'collectionId');
+                $content = $this->removeRules($content);
+                $content = $this->addCollectionPermissionLevel($content);
+                break;
+            case "database.updateCollection":
+                $content = $this->removeRules($content);
+                $content = $this->addCollectionPermissionLevel($content);
+                break;
+        }
+
+        return $content;
+    }
+
+    // New parameters
+
+    protected function addId(array $content, string $key): array
+    {
+        $content[$key] = 'unique()';
+        return $content;
+    }
+
+    protected function addCollectionPermissionLevel(array $content): array
+    {
+        $content['permission'] = 'document';
+        return $content;
+    }
+
+    // Deprecated parameters
+
+    protected function removeRules(array $content): array
+    {
+        unset($content['rules']);
+        return $content;
+    }
+
+    protected function removeOrderType(array $content): array
+    {
+        unset($content['orderType']);
+        return $content;
+    }
+
+    protected function removeOrderCast(array $content): array
+    {
+        unset($content['orderCast']);
+        return $content;
+    }
+
+    protected function removeParentProperties(array $content): array
+    {
+        unset($content['parentDocument']);
+        unset($content['parentProperty']);
+        unset($content['parentPropertyType']);
+        return $content;
+    }
+
+    // Modified parameters
+
+    protected function convertStatus(array $content): array
+    {
+        $content['status'] = $content['status'] === 2 ? false : true;
+        return $content;
+    }
+
+    protected function convertOrder(array $content): array
+    {
+        $content['orderAttributes'] = [ $content['orderField'] ];
+        $content['orderTypes'] = [ $content['orderType'] ];
+
+        unset($content['orderField']);
+        unset($content['orderType']);
+
+        return $content;
+    }
+
+    protected function convertQueries(array $content): array
+    {
+        $queries = [];
+
+        if(!empty($content['filters'])) {
+            foreach ($content['filters'] as $filter) {
+                $operators = ['=' => 'equal', '!=' => 'notEqual', '>' => 'greater', '<' => 'lesser', '<=' => 'lesserEqual', '>=' => 'greaterEqual'];
+                foreach ($operators as $operator => $operatorVerbose) {
+                    if (\str_contains($filter, $operator)) {
+                        $usedOperator = $operator;
+                        break;
+                    }
+                }
+
+                if(isset($usedOperator)) {
+                    [ $attributeKey, $filterValue ] = \explode($usedOperator, $filter);
+                    // TODO: String or not? Any way to figure out?
```

Comment:
Not sure I understand

Reply:
every value is currentli like `"this"`. So it ould be `name.equal("Matej")`, which is fine. But it will also be `age.greater('12')`, which should probably not be a string, instead, only `age.greater(12)`.